js/utility/flex-detect.js

```diff
@@ -1,8 +1,10 @@
-namespacer('seniorExpo.utility');
+namespacer('bcpl.utility');
 
-seniorExpo.utility.flexDetect = ((document, $) => {
-	const init = () => {
-		const hasFlex = document.createElement('div').style.flex !== undefined;
+bcpl.utility.flexDetect = ((document, $) => {
+	const init = (testDoc) => {
+		const actualDoc = testDoc || document;
+
+		const hasFlex = actualDoc.createElement('div').style.flex !== undefined;
 
 		if (!hasFlex) {
```

Comment:
is this missing the end bracket for the if?

Reply:
Nope. You may have to point out what you mean, but all of the brackets are there.